### DISCUSSION TITLE
Couldn't connect to a websocket

### DISCUSSION THREAD

**Discussion by gunthrek** [Kounex/obs_station#70] [Q&A]
Hello!

I'm trying to connect to my OBS instance using Blade, but no matter what I try I keep getting the "couldn't connect to a websocket" error.  I used the QR code as well as specific passwords, ensured everything is on port 4455, all to no avail.  There's another OBS controller app from the Google Play store that connects just fine, so I'm not sure it's a network/hardware problem.  What could be some reasons Blade fails to connect?

**Comment by dodgeit1337** [comment]
You need to switch the network connection type in Windows from Public Network to Private Network, this helped me.
